datahub-graphql-core/src/main/resources/gms.graphql

```diff
@@ -2908,3 +2926,36 @@ enum TimeRange {
     YEAR
     ALL
 }
+
+type DatasetFieldProfile {
+	fieldPath: String!
+	uniqueCount: Long
+	uniqueProportion: Float
+	nullCount: Long
+	nullProportion: Float
+	min: String
+	max: String
+	mean: String
+	median: String
+	stdev: String
+	sampleValues: [String!]
+}
+
+type TimeWindow {
+  startTimeMillis: Long!
+  durationMillis: Long!
+}
+
+type PartitionSpec {
+  partition: String!
+  timePartition: TimeWindow
+}
+
+type DatasetProfile implements TimeSeriesAspect {
+  timestampMillis: Long!
```

Comment:
so even if you implement an interface, do you still need to declare this?

Reply:
Unfortunately, yes